_src/pattern-library/sass/components/_layouts.scss

```diff
@@ -0,0 +1,700 @@
+// ------------------------------
+// edX Pattern Library: Components - Layouts
+
+// About: Contains base settings and options for the preset default Pattern Library UI Layouts.
+// ----------------------------
+
+// #CONFIG
+// #GLOBALS
+// #UTILITY
+// #CANNED - IN VIEW
+// #CANNED - VIEW
+
+// ----------------------------
+
+// ----------------------------
+// #CONFIG
+// ----------------------------
+$susy-flush:            layout($susy-default inside); // set susy to make columns flush
+$susy-reversed:         layout($susy-default $grid-direction-reversed); // set susy to make columns display in reverse order
+$susy-view:             layout($susy-default auto); // set susy to make columns view display in reverse order
+$susy-view-flush:       layout($susy-default auto inside); // set susy to make view columns display in reverse order
+$susy-view-reversed:    layout($susy-default auto $grid-direction-reversed); // set susy to make view columns display in reverse order
+
+// ----------------------------
+// #GLOBALS
+// ----------------------------
+
+
+// ----------------------------
+// #UTILITY
+// ----------------------------
+
+// CASE: Layout with gutter-separated columns
+%layout-default {
+    @include container($susy-default);
+}
+
+// CASE: Layout with flush columns
+%layout-flush {
+    @include container($susy-flush);
+}
+
+// CASE: Layout with reversed columns
+%layout-reversed {
+    @include container($susy-reversed);
+}
+
+// CASE: View-based layout
```

Comment:
@frrrances, let me know what you think about this naming convention for these types of layouts (once that take up the entire viewport's width. I imagined that these would be used for complete view layouts or at the very most for specific global elements (headers, bottom utility navigation, etc.). Happy to figure out a better term than ``view`` if you think its not clear.